src/store.js

```diff
@@ -44,7 +44,18 @@ export const store = reactive({
     template: '',
     include_test: false,
     output_dir: './logs',
-    log_every_iters: 10
+    log_every_iters: 10,
+    nproc_per_node: 2,
+    nnodes: 1,
+    master_addr: '127.0.0.1',
```

Comment:
It actually doesn't matter, since these params are set by default when we click on the "Training" tab if we see the `__DEV_CONFIG__.json` and if we inspect the `metadata.json`. So we can avoid filling the default config with these params, but they would still be populated at runtime. https://github.com/pytorch-ignite/code-generator/blob/def2c4686da003e6180a075189cff33e6cd10246/src/metadata/metadata.json#L36